sniffr/breeds/breeds.csv

```diff
@@ -0,0 +1,151 @@
+Breed,height_low_inches,height_high_inches,weight_low_lbs,weight_high_lbs

```

Comment:
Good list to start with.
I'm guessing we'll need to cover some dogs that aren't official show varieties (or an endpoint to add more later).
Some notable missing ones from the list: pit bull, dingo, labradoodle, a variety of mixed breeds, and maybe a catch-all category for "mutt."

Reply:
yeah, true, ;_; will have to find a more legit, list. Those are like really notable.

Reply:
Yeah :/ the AKC is only fancy purebred breeds. Labradoodles are a mixed breed so they wouldn't be included in most standard breed lists.